core/cli/src/help.ts

```diff
@@ -50,7 +50,7 @@ ${(Hook.constructor as any).description ? (Hook.constructor as any).description
 ${
   tasks && tasks.tasks.length
     ? `runs ${tasks.tasks.length > 1 ? 'these tasks' : 'this task'}:
-${tasks.tasks.map((task) => `- ${styles.task(task)}`).join('\n')}`
+${tasks.tasks.map((task) => `- ${styles.task(task.task)}`).join('\n')}`
```

Comment:
**suggestion**: can we maybe refactor this so its not `tasks.tasks`?

Reply:
hmm wow yeah this code is horrible to read 😂 i'll see what i can do! although tbh, the code for these messages isn't super important, i'm not particularly fussed if they're a bit clunky

Reply:
actually we need to rework `--help` anyway so i'll leave this for now